prosody/Dockerfile

```diff
@@ -1,10 +1,30 @@
 FROM jitsi/base
 
+ADD https://prosody.im/files/prosody-debian-packages.key /tmp/key
+
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/token/util.lib.lua /prosody-plugins/token/util.lib.lua
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/mod_token_verification.lua /prosody-plugins/mod_token_verification.lua
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/mod_auth_token.lua /prosody-plugins/mod_auth_token.lua
+
 RUN \
 	apt-dpkg-wrap apt-get update && \
-	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody && \
+	apt-dpkg-wrap apt-get install -y gnupg lua5.2 liblua5.2-dev lua-expat lua-socket lua-filesystem ssl-cert luarocks gcc libssl1.1 libssl1.0-dev git
```

Comment:
Please remove gnupg  (and I think all -dev libraries too) to keep the container smaller.

Reply:
Gnupg is needed to add the key and there was another problem with the upgrade to lua 5.2 so that I had to add the dev package.
I can try it without them again.
Gnupg can be removed after adding the key.

Reply:
Sorry, I meant remove them at the end, after the installation is done :-) They are probably needed at build time indeed.

Reply:
Alright. It's pushed.